packages/test/test-service-load/src/loadTestDataStore.ts

```diff
@@ -554,6 +558,11 @@ class LoadTestDataStore extends DataObject implements ILoadTest {
 		const opsSendType = config.testConfig.opsSendType ?? "staggeredReadWrite";
 		const opsPerCycle = (config.testConfig.opRatePerMin * cycleMs) / 60000;
 		const opsGapMs = cycleMs / opsPerCycle;
+		const futureOpPeriod =
```

Comment:
#nit I would separate changes to the stress tests from the rest. Makes the PR smaller + easier to rollback in case things go badly.

Reply:
I'm going to leave it all together, it's all just tests (which I wouldn't expect to be at risk for regressing during the time they were reverted).  In that worst-case scenario, I could check in just the non-stress tests or something anyway.